common/src/test/java/com/revenuecat/purchases/common/verification/DefaultSignatureVerifierTest.kt

```diff
@@ -0,0 +1,53 @@
+package com.revenuecat.purchases.common.verification
+
+import android.util.Base64
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.annotation.Config
+
+@RunWith(AndroidJUnit4::class)
+@Config(manifest = Config.NONE)
+class DefaultSignatureVerifierTest {
+
+    private val testPublicKey = "WEvg1j3R0lppwHRfLup/lzYZMQMEj5csDHeQe/YP1p4="
```

Comment:
I generated this manually for the tests. I could use Tink to generate new private/public keys for tests on the fly, but this seemed more deterministic, otherwise we would just be testing the library.